.taskcluster.yml

```diff
@@ -226,13 +223,9 @@ tasks:
                                     TEMPLATE_HEAD_REPOSITORY: '${template.repo}'
                                     TEMPLATE_HEAD_REV: '${template.branch}'
                                     TEMPLATE_HEAD_REF: '${template.branch}'
+                                    TEMPLATE_PIP_REQUIREMENTS: taskcluster/requirements.txt
```

Comment:
This should be `XPI`, not `TEMPLATE` :)

Reply:
Actually nope :)

XPI will refer to the xpi repo itself which won't have `requirements.txt` in it. So it will need to install it from the TEMPLATE repo instead.